mkdocs/docs/development.md

```
@@ -73,8 +106,170 @@ use:
+```
+
+You can also run tests in parallel using `parallel`:
+
+```bash
+sudo apt install parallel
+parallel behave ::: $(echo $(find features -type f -name '\*.feature'))
+```
+
+or use `behavex` to speedup tests execution:
+
+- https://github.com/hrcorval/behavex
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)